connectors/email/src/main/java/io/camunda/connector/email/client/jakarta/utils/JakartaUtils.java

```diff
@@ -321,7 +321,7 @@ public void moveMessage(Store store, Message message, String targetFolder) {
       char separator = imapFolder.getSeparator();
       String targetFolderFormatted =
           Optional.ofNullable(targetFolder)
-              .map(string -> string.split("\\."))
+              .map(string -> string.split("[./]"))
```

Comment:
... and use this constant here as well?

Or create a simple static method for this logic. Would be even better